### ISSUE TITLE
revision of example in <byline>

### ISSUE THREAD

**Issue by martinascholger** [TEIC/TEI#2087]
This issue was spotted by @martindholmes (
[https://lists.tei-c.org/pipermail/tei-council/2020-November/026587.html](https://lists.tei-c.org/pipermail/tei-council/2020-November/026587.html)) when reviewing the "copia" examples. The last example of the `<byline>` specs in French needs revision. 

The comma should be outside the `<docAuthor>`, and there should be a space after docteur.

```
<byline>Par le docteur<docAuthor>Martin Becanus,</docAuthor> au tome 
troisiesme de ses
  opuscules</byline>
```

**Comment by lb42** [comment]
Could also argue for including "le docteur" as part of the docAuthor

**Comment by HelenaSabel** [comment]
Typo corrected with 86dc263.
The contents of the named entity in relation to “le docteur” were not explicitly addressed.
